src/megaapi_impl.cpp

```diff
@@ -31314,6 +31368,35 @@ void MegaHandleListPrivate::addMegaHandle(MegaHandle h)
     mList.push_back(h);
 }
 
+MegaIntegerListPrivate::MegaIntegerListPrivate(const vector<int64_t> &integers)
+    : mIntegers(integers)
+{
+
+}
+
+MegaIntegerListPrivate::~MegaIntegerListPrivate()
+{
+
+}
+
+MegaIntegerList* MegaIntegerListPrivate::copy() const
+{
+    return new MegaIntegerListPrivate(mIntegers);
+}
+
+int64_t MegaIntegerListPrivate::get(int i) const
+{
+    if (i >= static_cast<int>(mIntegers.size()))
+        return -1;
```

Comment:
```suggestion
        {
                return -1;
        }
```

Reply:
Already agreed that `if`s and alike, when followed by single-line, can be left without curly brackets. I also prefer to add them, but not mandatory ;)